doc/source/ray-overview/installation.rst

```diff
@@ -28,6 +31,32 @@ To install Ray libraries:
   pip install -U "ray[rllib]"  # installs Ray + dependencies for Ray RLlib
   pip install -U "ray[serve]"  # installs Ray + dependencies for Ray Serve
 
+From conda-forge
+~~~~~~~~~~~~~~~~
+Ray can also be installed as a conda package on linux and windows
+
+.. code-block:: bash
+
+  conda create -c conda-forge python=3.9 -n ray_dev
+  conda activate ray_dev
+
+  # Install Ray with support for the dashboard + cluster launcher
+  conda install -c condaforge "ray[default]"
```

Comment:
```suggestion
  # Install Ray with support for the dashboard + cluster launcher
  conda install -c conda-forge "ray-default"
```

Conda doesn't support the `[optional]` syntax, which is why we need to use separate outputs.